front/src/applications/stdcm/components/StdcmResults/StdcmResults.tsx

```diff
@@ -93,6 +98,57 @@ const StdcmResults = ({
     return extractMarkersInfo(outputs.results.simulationPathSteps);
   }, [hasSimulationResults, outputs]);
 
+  const [refSchedule, setRefSchedules] = useState<RefSchedule[]>([]);
+
+  const [postRefSchedules] = osrdEditoastApi.endpoints.postSimilarTrains.useMutation();
+  const requestTrainSchedules = async () => {
+    const { consist, pathSteps } = selectedSimulation.inputs;
+    if (!consist) {
+      return;
+    }
+
+    const key = (uic: number, ch: string) => `${uic}-${ch}`;
+
+    const stops = pathSteps.reduce((acc, ps) => {
+      const k = key(ps.location?.uic ?? -1, ps.location?.secondary_code ?? '');
+      acc.set(k, !ps.isVia || (ps.stopFor !== undefined && ps.stopFor.ms > 0));
+      return acc;
+    }, new Map<string, boolean>());
+
+    const waypoints =
+      selectedSimulation.outputs?.pathProperties.manchetteOperationalPoints?.map((op) => {
+        const k = key(op.extensions?.identifier?.uic ?? -1, op.extensions?.sncf?.ch ?? '');
+        return {
+          id: op.opId!,
+          stop: stops.get(k) ?? false,
+        };
+      }) ?? [];
+
+    const request = {
+      infra_id: infraId,
+      rolling_stock: {
+        name: consist.tractionEngine?.name ?? '',
+        speed_limit_tag: consist.speedLimitByTag,
+      },
+      timetable_id: timetableId,
+      waypoints,
+    };
+
+    const response = await postRefSchedules({ body: request });
+    const rawSchedules = response.data?.similar_trains ?? [];
+
+    const enrichedSchedules = addSecondaryCodesToRefSchedules(rawSchedules, pathSteps);
+
+    setRefSchedules(enrichedSchedules);
+  };
+
+  useEffect(() => {
+    if (isSelectedSimulationRetained) {
+      requestTrainSchedules();
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Hum you don't need this eslint disable since this rule is disabled in our config